scrapli_community/huawei/smartax/async_driver.py

```diff
@@ -41,6 +41,8 @@ async def default_async_on_close(conn: AsyncNetworkDriver) -> None:
     await conn.acquire_priv(desired_priv=conn.default_desired_privilege_level)
     conn.channel.write(channel_input="quit")
     conn.channel.send_return()
+    conn.channel.write(channel_input="y")
```

Comment:
is this common/universal for the platform or is it like a newer thing that wasn't handled before? I dont know that it matters a ton since this can fail silently and likely wouldn't break anything, just curious mostly :)

Reply:
cool, sounds good! im not too worried about this one since it shouldn't really matter much anyway but I was just curious 😎